CoEpi/ui/alerts/AlertDetailsViewController.swift

```diff
@@ -32,37 +41,43 @@ struct AlertDetailsView: View {
     var body: some View {
         NavigationView {
             VStack(alignment: .leading, spacing: 4) {
-
                 Text(alertViewData.title)
                     .font(.system(size: 28))
                     .fontWeight(.bold)
                     .foregroundColor(Color(UIColor.coEpiPurple))
-
                 Text(alertViewData.reportTime)
                     .font(.system(size: 13))
                     .fontWeight(.light)
-
                 Text(alertViewData.contactDuration)
                     .font(.system(size: 17))
                     .fontWeight(.semibold)
                     .padding(.top, 4)
-
                 Text(alertViewData.avgDistance)
                     .font(.system(size: 17))
                     .fontWeight(.semibold)
-
                 Text(alertViewData.minDistance)
                     .font(.system(size: 17))
                     .fontWeight(.semibold)
-
                 Text(alertViewData.symptoms)
                     .font(.system(size: 13))
                     .fontWeight(.medium)
                     .padding(.top, 4)
-
             }
             .padding(EdgeInsets(top: 20, leading: 38, bottom: 20, trailing: 38))
             .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .topLeading)
         }
+        .actionSheet(isPresented: $viewModel.showingActionSheet) {
+            ActionSheet(
+                title: Text(L10n.Alerts.Details.More.title),
+                buttons: [
+                    .default(Text(L10n.Alerts.Details.More.delete)) {
+                        self.viewModel.delete()
```

Comment:
"delete" operation should be  ".destructive", right @i-schuetz  ?

Reply:
Normally yes, but that makes the label red, and in the designs it's black ("Report a problem" is red).